doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -58,6 +58,13 @@ Protocols
 This section provides detailed lists of changes by :ref:`protocol <protocols>`.
 See `Samples`_ for lists of changes for the protocol-related samples.
 
+Bluetooth mesh
+--------------
+
+* Added:
+  * :c:struct:`bt_mesh_sensor_srv` context to relevant callbacks and APIs to help resolve the associated sensor model instance.
```

Comment:
Please add space between Added: and the next bullet, I am sure it does not render correctly. Not sure how I missed this...

Reply:
Ok. Are these guidelines written somewhere?

Reply:
Yes, you can have a look here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/contribute/documentation/index.html#lists
General guidelines from Zephyr: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/contribute/documentation/index.html#doc-guidelines
And some in NCS: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#

Reply:
Doc build is broken and it looks like an indentation issue again. I'll push a commit :)